raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -91,6 +92,28 @@ private boolean updateHighWatermark() {
         return false;
     }
 
+    private OptionalLong updateLastFetchTimestamp() {
```

Comment:
nit: we probably need an illegal state exception here when this is a single quorum.

Reply:
This function doesn't update anything IIUC.